src/test/kotlin/no/nav/syfo/sykmelding/service/SendtSykmeldingServiceTest.kt

```diff
@@ -89,55 +90,58 @@ class SendtSykmeldingServiceTest :
                 storedSykmeldingsperioderAfterDelete.size shouldBe 0
             }
 
-            it("Should return true for active sendt sykmelding if period exists between fom and tom") {
+            it("Should return true for active sendt sykmelding if period exists between fom and tom plus 16 days") {
                 val sykmeldingId = "123"
                 val orgnumber = "456"
                 val employeeIdentificationNumber = "789"
                 val sykmeldingsperioder =
                     listOf(
                         SykmeldingsperiodeAGDTO(
-                            fom = LocalDate.now().minusWeeks(10),
-                            tom = LocalDate.now().plusWeeks(10),
+                            fom = LocalDate.now().minusDays(25),
+                            tom = LocalDate.now().minusDays(15),
```

Comment:
Er det mening at vi tester `minusDays(15)` og ikke `plusDays(15)` ?

Reply:
Ja, for å teste at en sykmelding som er "gått ut" for 15 dager siden er gyldig som "aktiv sykmelding" :)

Reply:
På den måten kan en sykmeldt ha utgått sykmelding, men likevel få lov å sende inn oppfølgingsplan inntil 16 (15) dager